java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -384,6 +384,38 @@ public ColumnVector isNull() {
     return new ColumnVector(isNullNative(getNativeView()));
   }
 
+  /**
+   * Returns a Boolean vector with the same number of rows as this instance, that has
+   * TRUE for any entry that is an integer, and FALSE if its not an integer
+   *
+   * NOTE: Integer doesn't mean a 32-bit integer. It means a number that is not a fraction.
+   * i.e. If this method returns true for a value could still result in an overflow or underflow.
+   *
+   * Also note that exponents are handled but might not be what the user expects. This method will
+   * convert the mantissa without regarding the exponent before ending the conversion when it hits
+   * the decimal. e.g. 1.2e-4 => 1 but 0.00012 => 0
+   *
+   * @return - Boolean vector
+   */
+  public ColumnVector isInteger() {
+    assert type == DType.STRING;
+    return new ColumnVector(isInteger(getNativeView()));
+  }
+
+  /**
+   * Returns a Boolean vector with the same number of rows as this instance, that has
+   * TRUE for any entry that is a float, and FALSE if its not a float
+   *
+   * NOTE: Float doesn't mean a 32-bit float. It means a number that is a fraction.
```

Comment:
This claims it will only return true for fractional values but the unit tests show that it will return true for integral values as well.  If this will return true for a superset of the values that would return true  for `isInteger`, it would be helpful to document that here.